### ISSUE TITLE
Searching for objects in belongs_to associations

### ISSUE THREAD

**Issue by fabianof** [shioyama/mobility-ransack#18]
Isn't that possible or am I missing something? For instance:

```
class Athlete < ApplicationRecord
	belongs_to :playing_position
end

class PlayingPosition < ApplicationRecord
	extend Mobility
	
	translates :name
end
```

pry(main)> Athlete.ransack(playing_position_name_cont: 'pont').result.size
   (0.7ms)  SELECT COUNT(*) FROM `athletes` LEFT OUTER JOIN `playing_positions` ON `playing_positions`.`id` = `athletes`.`playing_position_id` WHERE `PlayingPosition_name_pt_br_string_translations`.`value` LIKE '%pont%'
ActiveRecord::StatementInvalid: Mysql2::Error: Unknown column 'PlayingPosition_name_pt_br_string_translations.value' in 'where clause'
from /Users/fabiano/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query'
Caused by Mysql2::Error: Unknown column 'PlayingPosition_name_pt_br_string_translations.value' in 'where clause'
from /Users/fabiano/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query'

**Comment by shioyama** [comment]
Sorry I just saw this. I'll try to reproduce and have a look at #19.

**Comment by shioyama** [comment]
Fix released in 1.1.0

**Comment by fabianof** [comment]
Thanks a lot, @shioyama!
